enhancements/service-catalog/service-catalog-and-broker-removal.md

```diff
@@ -103,53 +100,53 @@ would be removed:
 The idea is that the Job will run during the upgrade until completion. Once
 finished, the Service Catalog will have been removed from the cluster.
 
-The Service Catalog operators would *remain* in the release payload but the
+The Service Catalog operators would remain in the release payload but the
 Service Catalog itself would be removed.
 
 There are a few alternatives that are discussed in the Alternatives section of
 this proposal, please see below.
 
 #### Story 2
 
-Implement a mechanism that ensures Service Brokers are not present after
-an upgrade to OpenShift 4.4.
-
-Create a new CVO managed operator with a Job definition as in Story 1 to remove
-the Ansible Service Broker and Template Service Broker. The new CVO operator is
-needed because the Ansible Service Broker operator and the Template Service
-Broker operator are both OLM managed operators which may or may not be present
-during cluster upgrades.
-
-NOTE: this would add a new item to the OpenShift 4.4 release payload.
-
-There is also an alternative to this approach in the Alternatives section below.
+Keep the Service Catalog _Operators_ in the OpenShift 4.4 release payload.
 
 #### Story 3
 
-1. Keep the Service Catalog _Operators_ in the OpenShift 4.4 release payload
-1. Remove the Service Catalog from the OpenShift 4.4 release payload (ART)
-
-#### Story 4
-
 Remove the Brokers from productization
 
 1. Remove the Template Service Broker from productization (ART)
 1. Remove the Ansible Service Broker from productization (ART)
 1. Remove the broker sections from the documentation (Docs)
 
-#### Story 5
+#### Story 4
 
 Remove the broker operators from productization
 
 1. Remove the Template Service Broker Operator from productization (ART)
 1. Remove the Ansible Service Broker Operator from productization (ART)
 1. Remove the operator instructions from the documentation (Docs)
 
-#### Story 6
+#### Story 5
 
 1. Remove the Service Catalog from productization (ART)
 1. Remove the installation instructions from the documentation (Docs)
 
+#### Story 6
+
+For OpenShift 4.3.z, update both Service Catalog Operators to set
+`Upgradeable = False`. This will cause z-stream upgrades to continue to work
```

Comment:
I think we need to be more clear here it will not always be false, it will be false when the service catalog is on. If you are in an unmanaged state or removed then it will be upgradable correct?

Reply:
Please re-review proposal. I have re-written a good portion of it to reflect our discussions.